dandiapi/api/views/asset.py

```diff
@@ -159,17 +159,9 @@ def get_queryset(self):
         self.raise_if_unauthorized()
         return super().get_queryset()
 
-    @swagger_auto_schema(
-        responses={
-            200: 'The asset metadata.',
-        },
-        manual_parameters=[ASSET_ID_PARAM],
-        operation_summary="Get an asset\'s metadata",
-        operation_description='',
-    )
```

Comment:
Why remove this?

Reply:
The single manual asset ID parameter was unnecessary, as it's already covered by the `lookup_field` and `lookup_value_regex` class fields. However, I somehow overlooked the `responses` and `operation_summary` fields, so I'll add those back.